src/website/views/components/alert/_examples.pug

```diff
@@ -552,6 +552,94 @@ p.-text
           </button>
         </div>
 
+h4 Closable alert interaction
+p.-text
+.example.-mb--3
+  .-p--3#interactive-wrapper
+    chi-button#alert-trigger-button(color='primary')
+      | Click me to show the alert
+    chi-alert.-mt--2#interactive-alert(title='Warning', color='warning', icon='warning', closable)
+      | This is a closable warning alert
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-interactive
+      li.-active
+        a(href=`#webcomponent-portal-interactive`) Web Component
+      li
+        a(href=`#html-portal-interactive`) HTML Blueprint
+  .chi-tabs-panel.-active#webcomponent-portal-interactive
+    .chi-tab__description
+      span.-text--grey
+        | This HTML example requires additional JavaScript.
+    :code(lang="html")
+      <!-- Wrapper -->
+      <div id="interactive-wrapper">
+        <!-- Trigger -->
+        <chi-button id="alert-trigger-button" color="primary">Click me to show the alert</chi-button>
+
+        <!-- Alert -->
+        <chi-alert id="interactive-alert" icon="circle-info" closable mutable>
+          This is a closable warning alert
+        </chi-alert>
+      </div>
+
+      <!-- Javascript -->
+      <script>
+        const interactiveAlert = document.getElementById('interactive-alert');
+        const interactiveAlertWrapper = document.getElementById('interactive-wrapper');
+        const interactiveAlertTrigger = document.getElementById('alert-trigger-button');
+
+        interactiveAlert.parentNode.removeChild(interactiveAlert);
+
+        interactiveAlertTrigger.addEventListener('chiClick', () => {
+          if (interactiveAlertWrapper.children.length === 1) {
+            interactiveAlertWrapper.appendChild(interactiveAlert.cloneNode(true));
+          }
+        })
+      </script>
```

Comment:
@mtadejalumen This logic is only needed for the example that we have in documentation. For developers the only needed logic is one button that triggers an alert. Please remove unnecessary code.